application/web/web.js

```diff
@@ -221,11 +240,25 @@ function saveCurrentEntry() {
     const newEntry = document.createElement('button');
     newEntry.innerText = title;
     const article = document.createElement('article');
-    article.setAttribute('id', new Date());
+
+    const date = new Date();
+    let day = date.getDate();
+    let month = date.getMonth() + 1;
+    let year = date.getFullYear();
+
+    newEntry.setAttribute('id', `${year}-${month}-${day}-title`);
+    article.setAttribute('id', `${year}-${month}-${day}`);
```

Comment:
can also remove or comment for the issue with implementing the Save Entry button